lesson_13_hm.py

```diff
@@ -0,0 +1,56 @@
+import json
+
+
+class DataStorage:
+    def __init__(self, file_path):
+        self.status = 'disconnect'
+        self.content = None
+        self._file_path = file_path
+
+    def _create_storage(self):
+        with open(self._file_path, 'w') as j_file:
+            json.dump([], j_file)
+
+    def connect(self):
+        try:
+            with open(self._file_path, 'r') as j_file:
+                self.content = json.load(j_file)
+                self.status = 'connected'

```

Comment:
If I catch exception - I try to return something, but here no.. why?
We have not return here and its confusing me 🤔